OrbitClientServices/ProcessClient.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitClientServices/ProcessClient.h"
+
+#include "OrbitBase/Logging.h"
+#include "grpcpp/grpcpp.h"
+#include "outcome.hpp"
+#include "services.grpc.pb.h"
+#include "symbol.pb.h"
+#include "tracepoint.pb.h"
+
+namespace {
+
+using orbit_grpc_protos::GetDebugInfoFileRequest;
+using orbit_grpc_protos::GetDebugInfoFileResponse;
+using orbit_grpc_protos::GetModuleListRequest;
+using orbit_grpc_protos::GetModuleListResponse;
+using orbit_grpc_protos::GetProcessListRequest;
+using orbit_grpc_protos::GetProcessListResponse;
+using orbit_grpc_protos::GetProcessMemoryRequest;
+using orbit_grpc_protos::GetProcessMemoryResponse;
+using orbit_grpc_protos::GetTracepointListRequest;
+using orbit_grpc_protos::GetTracepointListResponse;
+using orbit_grpc_protos::ModuleInfo;
+using orbit_grpc_protos::ProcessInfo;
+using orbit_grpc_protos::ProcessService;
+using orbit_grpc_protos::TracepointInfo;
+
+constexpr uint64_t kGrpcDefaultTimeoutMilliseconds = 1000;
+
+std::unique_ptr<grpc::ClientContext> CreateContext(uint64_t timeout_milliseconds) {
```

Comment:
nit: I think we do not use custom timeout anymore - maybe add = kGrpcDefaultTimeoutMilliseconds here and drop it from all calls below.

Reply:
Ok will do that, thanks!

Reply:
Done!